client/src/core/styles/_typography.scss

```diff
@@ -0,0 +1,157 @@
+//body text
+.typography__body {
```

Comment:
Weren't we going to put this in some kind of normalize stylesheet and apply this directly to the  `body` element?

Reply:
I think that is something we can look into but is not really in the scope of this change. This was only intended to be a quick update based on the things that have changed for the login page.